lms/models/assignment.py

```diff
@@ -81,6 +81,10 @@ class Assignment(CreatedUpdatedMixin, Base):
     groupings = association_proxy("assignment_grouping", "grouping")
     """List of groupings this assigments is related to."""
 
+    membership = sa.orm.relationship(
```

Comment:
Add new field to the assignments to get all "users" without having to write the query each time.

Reply:
`dynamic` makes this a query so que can filter by roles etc.